jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -484,7 +484,9 @@ private static DiagnosticProperties doJFR(String diagnosticCommand) {
 						Timer timer = new Timer();
 						TimerTask jfrDumpTask = new TimerTask() {
 							public void run() {
-								VM.stopJFR();
+								if (VM.isJFRRecordingStarted()) {
```

Comment:
Thid might be better done as the first thing in stopJFR to avoid this issue in the future.

Reply:
Yes, I added an equivalent check to the native